packages/core/upload/server/__tests__/bootstrap.test.js

```diff
@@ -35,7 +35,13 @@ describe('Upload plugin bootstrap function', () => {
         },
       },
       plugins: {
-        upload: {},
+        upload: {
+          services: {
+            metrics: {
+              startRegularMetricsUpdate() {},
```

Comment:
I'm a bit confused about the name of the method: what is an `irregular` metrics update? Should we rename this to `startScheduledMetricsUpdates` for example?

Reply:
scheduled works for me too ! `regular` means to me that it happens at a constant frequency. We have the same work in French. Maybe in english this sense is less used?
<img width="685" alt="Capture d’écran 2022-07-20 à 14 18 22" src="https://user-images.githubusercontent.com/11708220/179979899-63a6a78d-55b2-4eed-b399-361224e85402.png">

Reply:
It has the same literal meaning "we now return to our regularly scheduled programming" from old TV but in tech it doesn't carry the same weight. I'd suggest scheduled here also.